db-migrations/src/main/resources/db/migration/V1__initial-datamodel.sql

```diff
@@ -9,7 +9,14 @@ CREATE TABLE nav_ansatt
     personlig_ident varchar NOT NULL UNIQUE,
     telefonnummer   varchar,
     epost           varchar,
-    navn VARCHAR NOT NULL
```

Comment:
Fint at du merger scriptene. Jeg har også en pr med scriptendringer som jeg kan prøve å få ready for review